tests/phpunit/tests/theme.php

```diff
@@ -250,6 +250,36 @@ public function test_default_theme_matches_constant() {
 		$this->assertSame( $wp_default_theme_constant, $latest_default_theme->get_stylesheet(), 'WP_DEFAULT_THEME should match the latest default theme.' );
 	}
 
+	/**
+	 * Ensure that the default themes are included in the new bundled files.
+	 *
```

Comment:
Needs a `@ticket`

Reply:
Indeed that would be a bit yuck. I'll have a think if there's an alternative way that comes to mind.